src/Options.js

```diff
@@ -58,22 +68,69 @@ Options.prototype.extend = function(extension) {
     return new Options(data);
 };
 
+function sizeAtStyle(size, style) {
+    return style.size < 2 ? size : sizeStyleMap[size - 1][style.size - 1];
```

Comment:
For some reason I thought the sizes where started counting from `1`.  I think my reason for confusion is that I was conflating sizing command sizes with style command sizes.  :(  The fact that we're accessing values in `sizeStyleMap` should've clued me.